### PR TITLE
[ADD] rotordc_mail_activity_display_info

### PR DESCRIPTION
**Pull request by carmenbianca**
Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

## Odoo task (if applicable)

https://gestion.coopiteasy.be/web#id=8679&action=475&active_id=231&model=project.task&view_type=form&menu_id=

## Checklist before approval

- [ ] Tests are present (or not needed).
- [x] Credits/copyright have been changed correctly.
- [x] Change log snippet is present.
- [ ] (If a new module) Moving this to OCA has been considered.

### CODE REVIEW

File: rotordc_mail_activity_display_info/static/src/xml/activity.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        FIXME: It would be really nice if we could surgically add/remove
+        classes instead of copy-pasting an entire thing.
+        -->
+        <t t-jquery="a.o_activity_info" t-operation="attributes">
+            <!-- remove 'collapsed' -->
+            <attribute
+                name="class"
+            >btn btn-link btn-info text-muted o_activity_info ml4</attribute>
+            <attribute name="aria-expanded">true</attribute>
+        </t>
```
Comment (victor-champonnois) [inline rotordc_mail_activity_display_info/static/src/xml/activity.xml:14]: Have you tried to do it with this method? (I guess it works for templates?)
(and same for the one below adding 'show')
```
<xpath expr="//div[@id='node_id']" position="attributes">
    <attribute name="class" separator=" " add="d-none" />
</xpath>
```
from the link you sent me https://dev.to/opencode/don-t-break-your-and-others-odoo-xml-373i ^^

Comment (carmenbianca) [reply rotordc_mail_activity_display_info/static/src/xml/activity.xml:14]: I know, but this is a Python dictionary :( I have to remove the closing bracket, add the key-value pair I want, and re-add the closing bracket.

Comment (carmenbianca) [reply rotordc_mail_activity_display_info/static/src/xml/activity.xml:14]: Oh wait, I did the Python dictionary in another project—scratch that. I can't surgically remove classes, but I can try to do that for adding 'show' later in the file.

Comment (carmenbianca) [reply rotordc_mail_activity_display_info/static/src/xml/activity.xml:14]: Tried this; reverted in [aebb624](https://github.com/coopiteasy/cie-custom/pull/72/commits/aebb6247ab458ff9061a012b4d29b09c6ff41aa1)

It didn't work.

Comment (victor-champonnois) [reply rotordc_mail_activity_display_info/static/src/xml/activity.xml:14]: OK let's move on then :)

**Review by victor-champonnois — APPROVED** [review]

**Pull request merged**